WordPress/gutenberg
# Fix invisible Android activity indicator

* **Gutenberg Mobile:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/3716

## Description
Fixes wordpress-mobile/gutenberg-mobile#3708. Apply a patch for the React Native `ActivityIndicator` rendering invisible on Android. This applies the same patch that was previously applied to the Embed block to to React Native itself.

The core issue in React Native appears to be [wider reaching](https://git.io/JcNlc), but I chose to [limit the scope](https://git.io/JcNla) of the patch to the one issue we are currently facing (i.e. invisible activity indicators), as I was concerned applying a deeper, untested change might introduce regressions elsewhere.

This patch can be removed once we upgrade to a React Native version including fixes for the related issues: 

- https://github.com/facebook/react-native/issues/30056
- https://github.com/facebook/react-native/issues/29495
- https://github.com/facebook/react-native/issues/29378.

## How has this been tested?
1. Launched native Demo app on Android. 
2. Insert an Embed block. 
3. Set valid URL within the Embed block settings. 
4. Observe the activity indicator is visible. 

-AND- 

1. Launch native Demo app on Android. 
2. Add a Paragraph block. 
3. Tap the Link button. 
4. Enter text into the Link input to begin search for URLs. 
5. Observe the activity indicator visible. 

## Screenshots 

| Embed | Link |
| --- | --- |
| ![Android activity indicator](https://user-images.githubusercontent.com/438664/125134680-5d1c8600-e0cd-11eb-8beb-eff3b778fae2.jpg) | ![Android activity indicator link](https://user-images.githubusercontent.com/438664/125135457-9dc8cf00-e0ce-11eb-9695-a3941e0f4161.jpg) |

## Types of changes
Bug fix

## Checklist:
